tests/qt/qmetaobject/tst_qmetaobject.cpp

```diff
@@ -407,6 +374,39 @@ private slots:
 
     void notifySignalsInParentClass();
 
+public:
```

Comment:
Is this change also necessary?  I want to avoid diverging away from the upstream tests)  as it make merging more difficule.